gpu4pyscf/lib/gvhf-rys/gamma_inc.cu

```diff
@@ -0,0 +1,50 @@
+#include <float.h>
+#define LSUM_MAX        (LMAX*4)
```

Comment:
Is gamma_inc.cu the same as gamma_inc_unrolled.cu?

Reply:
No. they are different ways to assign threads.